python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -6076,7 +6076,13 @@ def test_corrwith(self):
     def _test_corrwith(self, psdf, psobj):
         pdf = psdf.to_pandas()
         pobj = psobj.to_pandas()
-        for method in ["pearson", "spearman", "kendall"]:
+        # Regression in pandas 1.5.0 when other is Series and method is "pearson" or "spearman"
+        # See https://github.com/pandas-dev/pandas/issues/48826
```

Comment:
FYI: This is the report to the pandas community to confirm if it officially regression in pandas.